tasks/nix.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
+if [[ $EUID > 0 ]]
+then
+  echo "{\"Error\": \"Please run as root or sudo\"}"
```

Comment:
This error could be ambiguous e.g. What should I run as root? bolt? the task?  Perhaps give the user more information about how to fix this.  The sudo password stuff is at https://puppet.com/docs/bolt/latest/bolt_configuration_options.html#ssh-transport-configuration-options

Reply:
I have update the error message. Please have a look.

Reply:
Changes look good!  @clairecadman Did you have any comments on error message?